### DISCUSSION TITLE
generate command does not return/close on success

### DISCUSSION THREAD

**Discussion by crumbolt** [nuxt/nuxt#23894] [Questions]
I am using nuxt3 to generate a static site. After running npm run generate the process completes and outputs: 

[nitro] ✔ You can preview this build using npx serve .output/public
✔ You can now deploy .output/public to any static hosting!

I am trying to use this in an automated setup but the deployment process stalls at this point as the generate command never returns. Is there a setting/option that will force the command to terminate on success?

**Comment by surgiie** [comment]
Im also noticing this issue. It just stalls at times and is holding up our github actions and causing really long running github action jobs. Any solution to this?

**Comment by TheAlexLichter** [comment]
This is usually caused by a module which registers a file watcher but not cleaning it up when `close` hook is called.
